src/components/ColorSelector/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { forwardRef, memo } from "react";
+import styles from "./styles.module.css";
+import { motion } from "framer-motion";
+
+const ColorSelector = memo(
```

Comment:
Not bad lo de usar memo, pero te recomiendo leerte esta nota en la documentación de React que habla de lo llamado "optimización prematura": https://react.dev/reference/react/memo#should-you-add-memo-everywhere